modules/conflict_resolver/php/resolved_conflicts.class.inc

```diff
@@ -197,40 +159,48 @@ class Resolved_Conflicts extends \NDB_Menu_Filter
         if ($user->hasPermission('access_all_profiles')) {
             // get the list of study sites - to be replaced by the Site object
             $sites = \Utility::getSiteList();
+            foreach ($sites as $key => $site) {
+                unset($sites[$key]);
+                $sites[$site] = $site;
+            }
             if (is_array($sites)) {
                 $sites = array('' => 'All') + $sites;
             }
         } else {
             // allow only to view own site data
             $sites = $user->getStudySites();
+            foreach ($sites as $key => $site) {
+                unset($sites[$key]);
+                $sites[$site] = $site;
+            }
             $sites = array('' => 'All User Sites') + $sites;
         }
 
         // Add form elements
-        $this->addSelect('site', 'Site:', $sites);
-        $this->addSelect('Instrument', 'Instrument:', $instruments);
-        $this->addSelect('visit', 'Visit label:', $visits);
+        $this->addSelect('site', 'For Site:', $sites);
+        $this->addSelect('instrument', 'Instrument:', $instruments);
+        $this->addSelect('visitLabel', 'Visit label:', $visits);
         $this->addBasicText(
-            'CandID',
-            'DCCID:',
+            'candID',
+            'CandID:',
             array(
-             "size"      => 9,
-             "maxlength" => 6,
+             'size'      => 9,
+             'maxlength' => 6,
             )
         );
         $this->addBasicText(
-            'PSCID',
+            'pSCID',
             'PSCID:',
             array(
-             "size"      => 9,
-             "maxlength" => 7,
+             'size'      => 9,
+             'maxlength' => 7,
             )
         );
-        $this->addBasicText('Question', 'Search keyword:');
-        $this->addBasicText('ResolutionTimestamp', 'Resolution Timestamp:');
+        $this->addBasicText('question', 'Question:');
+        $this->addBasicText('timestamp', 'Resolution Timestamp:');
```

Comment:
Change to the following to fix Resolution Timestamp filter:
$this->addBasicText('resolutionTimestamp', 'Resolution Timestamp:');